package.json

```diff
@@ -27,56 +28,53 @@
   },
   "dependencies": {
     "nuxt": "3.0.0-rc.3",
-    "unplugin-icons": "^0.14.3",
-    "unplugin-vue-components": "^0.19.6"
+    "unplugin-icons": "^0.14.5",
+    "unplugin-vue-components": "^0.19.9"
   },
   "devDependencies": {
-    "@nuxt/test-utils-edge": "3.0.0-rc.3-27550969.a4a3cff",
+    "@nuxt/test-utils-edge": "^3.0.0-rc.4-27600153.ad42023",
     "@nuxt/types": "^2.15.8",
     "@nuxt/typescript-build": "^2.1.0",
-    "@nuxt/vite-builder": "3.0.0-rc.3",
+    "@nuxt/vite-builder": "3.0.0-rc.4",
     "@nuxtjs/eslint-config-typescript": "^10.0.0",
     "@nuxtjs/eslint-module": "^3.1.0",
-    "@nuxtjs/tailwindcss": "^5.1.2",
-    "@testing-library/dom": "^8.13.0",
+    "@nuxtjs/tailwindcss": "^5.1.3",
+    "@testing-library/dom": "^8.14.0",
     "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/user-event": "^14.2.0",
-    "@testing-library/vue": "^6.4.2",
+    "@testing-library/user-event": "^14.2.1",
+    "@testing-library/vue": "^6.6.0",
     "@types/dotenv-safe": "^8.1.2",
-    "@types/jest": "^27.5.1",
-    "@types/node": "^17.0.35",
-    "@typescript-eslint/eslint-plugin": "^5.26.0",
-    "@typescript-eslint/parser": "^5.26.0",
+    "@types/eslint": "^8.4.3",
+    "@types/jest": "^28.1.3",
+    "@types/node": "^18.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.29.0",
+    "@typescript-eslint/parser": "^5.29.0",
     "@vitejs/plugin-vue": "^2.3.3",
     "@vitejs/plugin-vue-jsx": "^1.3.10",
-    "@vue/compiler-sfc": "^3.2.36",
+    "@vue/compiler-sfc": "^3.2.37",
+    "@vue/eslint-config-typescript": "^11.0.0",
     "@vue/test-utils": "^2.0.0",
     "dotenv-safe": "^8.2.0",
-    "eslint": "^8.16.0",
+    "eslint": "^8.18.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-nuxt": "^3.2.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-tailwindcss": "^3.5.0",
-    "eslint-plugin-vue": "^9.0.1",
-    "eslint-plugin-vuejs-accessibility": "^1.1.1",
+    "eslint-plugin-tailwindcss": "^3.5.2",
+    "eslint-plugin-vue": "^9.1.1",
+    "eslint-plugin-vuejs-accessibility": "^1.2.0",
     "husky": "^8.0.1",
-    "jsdom": "^19.0.0",
+    "jsdom": "^20.0.0",
     "postcss": "^8.4.14",
-    "prettier": "^2.6.2",
+    "prettier": "^2.7.1",
     "pretty-quick": "^3.1.3",
     "swagger-typescript-api": "^9.3.1",
-    "tailwindcss": "^3.0.24",
-    "typescript": "^4.7.2",
-    "vite": "^2.9.9",
-    "vitest": "^0.12.9",
+    "tailwindcss": "^3.1.4",
+    "typescript": "^4.7.4",
+    "vite": "^2.9.12",
+    "vite-plugin-eslint": "^1.6.1",
+    "vitest": "^0.16.0",
     "vitest-fetch-mock": "^0.1.0",
-    "vue": "3.2.36",
-    "webpack": "^5.72.1"
-  },
-  "peerDepsInDevDeps": {
-    "@vue/compiler-sfc": "^3.2.36",
-    "vite": "^2.9.9",
-    "vue": "3.2.36",
-    "webpack": "^5.72.1"
```

Comment:
💭 Is there a reason as to why we're removing the `peerDepsInDevDeps` section now?

Reply:
I'm not sure how useful it is and may be abit confusing for others.